openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -115,7 +115,8 @@ <h3 class="message__title">{% trans "We could not save your progress" %}</h3>
                             </li>
                             {% if allow_latex %}
                                 <li class="list--actions__item">
-                                    <button type="submit" class="submission__preview  action action--save" aria-describedby="response__preview_explanation__{{ xblock_id }}">
+                                    <button type="submit" class="submission__preview  action action--save" aria-describedby="response__preview_explanation__{{ xblock_id }}"
```

Comment:
FYI, before my previous PR, the preview button was "disabled" always upon initial rendering. I removed that because I thought you should be able to preview your response as long as you had non-empty text, but I didn't add anything to make sure that it was disabled if there was no text present.